connect/cli/plugins/project/extension/helpers.py

```diff
@@ -60,10 +63,14 @@ def bootstrap_extension_project(config, data_dir, overwrite):
         else:
             ctx.update({var: answer})
 
-    renderer = BoilerplateRenderer(data_dir, ctx, overwrite)
-    project_dir = renderer.render()
+    renderer = BoilerplateRenderer(
+        ctx,
+        ['extension', 'templates', 'bootstrap'],
```

Comment:
A bit strange to pass the segments and than join them inside the BoilerplateRenderer.
IMO it's more clear to invoke with 'extension/templates/bootstrap'